modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -151,5 +165,18 @@ def enum_zones
       logger.debug "Enumerated authoritative dns zones: #{zones}"
       zones
     end
+
+    def enum_records(zone_name, node_name, type)
+      records = []
+      response = execute "/EnumRecords #{zone_name} #{node_name}.", "EnumRecords", true
+      response.each do |line|
+        logger.debug "/EnumRecords line=#{line}"
+        next unless line =~ /\s#{type}\s/
```

Comment:
you can pass in resource type to /enumrecords via /type option, in which case you won't need this line.